procrastinate/healthchecks.py

```diff
@@ -11,8 +12,8 @@ def __init__(self, job_store: store.BaseJobStore):
         )
         return result['check']
 
-    async def get_db_version_async(self):
+    async def check_db_version_async(self):
         result = await self.job_store.execute_query_one(
             query=sql.queries['get_latest_version'],
         )
-        return result['version']
+        return result['version'] == Migrator.version
```

Comment:
I wouldn't be opposed to leaving the `get_db_version_async` the way it was ?
Maybe have both ?
Also, maybe the check should rather raise than return False, so as to attach the details of the failure along with the result ?

Reply:
I changed into `get_schema_version_async`, without a try/except. It's now the cli that does the necessary check. 🙂